arangod/Cluster/FailureOracleFeature.cpp

```diff
@@ -213,7 +215,7 @@ void FailureOracleFeature::prepare() {
 void FailureOracleFeature::start() {
   TRI_ASSERT(_cache == nullptr);
   _cache = std::make_shared<FailureOracleImpl>(
-      server().getEnabledFeature<ClusterFeature>());
+      server().getFeature<ClusterFeature>());
```

Comment:
This change should not be necessary with any of the other two changes.

Reply:
Reverted